packages/atomic/src/components/result-template-components/atomic-result-link/atomic-result-link.tsx

```diff
@@ -17,10 +17,7 @@ export class AtomicResultValue {
   public render() {
     return (
       <a part="result-link" href={this.result.clickUri}>
-        <atomic-result-value
-          value="title"
-          shouldHighlightWith="titleHighlights"
-        ></atomic-result-value>
+        <atomic-result-text-value property="title"></atomic-result-text-value>
```

Comment:
Since we use `field` for the facet, and in the AdminUI when indexing, I think it would be more consistent to name the prop `field` too? Essentially trying to use the same name for the same concept.

Or is there a benefit of using a different name?